x-pack/plugins/stack_alerts/server/rule_types/geo_containment/executor.ts

```diff
@@ -82,7 +82,7 @@ export async function executor({
   const { activeEntities, inactiveEntities } = getEntitiesAndGenerateAlerts(
```

Comment:
Can we add
```
if (!services.alertsClient) {
  throw new AlertsClientError();
}
```
to the top of the executor? In the future when we remove the `alertFactory`, we can remove this check but for now, it will short circuit execution if something wasn't installed correctly in the alerts client